docs/blog/2024-07-25-nx-19-5-update.md

```diff
@@ -0,0 +1,176 @@
+
+Learn more about how you can [Parallelize Tasks Across Multiple Machines Using Nx Agents](/ci/intro/tutorials/github-actions#parallelize-tasks-across-multiple-machines-using-nx-agents):
+
+{% youtube
+src="https://youtu.be/0YxcxIR7QU0"
+title="Faster e2e Tests!"
+width="100%" /%}
+
+## Experimental: Gradle Test Atomization
+
+Our new [`@nx/gradle` plugin](/nx-api/gradle) now supports Test Atomization out of the box.
+
+![Gradle Atomized](/blog/images/2024-07-25/gradle-atomized.jpg)
+
+This means that as you add tests in your gradle projects, we'll automatically create a new task for each test class, allowing you to distribute the execution of these tasks in your CI pipeline, just like with our `@nx/playwright` and `@nx/cypress` plugins.
+
+## Support for React 19 (rc) and Angular 18.1
+
+Nx 19.5 adds support for the React 19 release candidate and updated our Angular package to support the latest Angular minor version: 18.1.
+
+When using our `@nx/angular` package, we'll automatically update you to the latest angular version when you run our migration. Using the `--interactive` flag you can choose to opt in or out of the latest Angular version:
+
+```shell
+> nx migrate latest --interactive
+✔ Do you want to update to TypeScript v5.5? (Y/n) · true
+✔ Do you want to update the Angular version to v18.1? (Y/n) · true
+Fetching @angular/core@18.1.2
+
+ NX   The migrate command has run successfully.
+```
+
+Our `@nx/react` package will now create new React applications using version 18.3, and we now support the new experimental `reactCompiler`. Users can install the [`babel-plugin-react-compiler` package](https://www.npmjs.com/package/babel-plugin-react-compiler) and you can follow [this guide for how to enable it with Nx](/recipes/react/react-compiler#react-compiler-with-nx).
+
+Note that due to the extent of breaking changes coming with React 19, we will not be providing a automated migration to React 19 via [`nx migrate`](/nx-api/nx/documents/migrate).
+
+## Automatically Update Nx
+
+As always - updating Nx and its plugins is easy as we ship an [automated migration command](/features/automate-updating-dependencies).
+
+```shell
+npx nx migrate latest
+```
+
+After updating your dependencies, run any necessary migrations.
+
+```shell
+npx nx migrate --run-migrations
+```
+
+## NEW: Nx Cloud Hobby Tier
+
+Our new expanded Hobby Tier now adds trial support for all features AND includes credits to run Nx Agents.
+
+Sample the entire suite of features to see its impact on your organization. Start with everything, scale when you need more.
+
+[![Nx Cloud Tiers](/blog/images/2024-07-25/start-with-everything-scale-when-you-need.jpg)](https://nx.app/pricing#plan-details)
+
+Checkout the [plan details page](https://nx.app/pricing#plan-details) for more info, and see how Nx Cloud can help you!
+
+## Monorepo World Conference Speakers to Be Announced Soon!!
```

Comment:
We need to adjust this. Ben just pushed out the first set of speakers to our website yesterday evening: https://monorepo.world